### ISSUE TITLE
make it available as a lazy version

### ISSUE THREAD

**Issue by benzkji** [fabiangermann/django-textblocks#12]
for use at module load time, for example as formfield labels. as a replacement/alternative to django's builtin ugettext_lazy.

I dont know that much about the lazy internals of django, so...I dont know how hard it is to implement? Any thoughts on this?

**Comment by matthiask** [comment]
(Untested! And Python 3-only because of `str` usage.)

```
from django.utils.functional import lazy
from textblocks.templatetags.textblock_tags import textblock

textblock_lazy = lazy(textblock, str)
```

**Comment by benzkji** [comment]
thx, and yes, I have it in use already more or less like this, it's easy, but needs to be tested ;-)

**Comment by fabiangermann** [comment]
Resolved with #36. Released with 0.7.
